prototype/technology.lua

```diff
@@ -1460,13 +1458,14 @@ data:extend(
                 },
                 prerequisites = { "laser-miners4", "neuro-miners3" },
                 unit = {
-                    count = 650,
+                    count = 1000,
                     ingredients = {
-                        { "automation-science-pack", 1 },
-                        { "logistic-science-pack", 1 },
-                        { "chemical-science-pack", 1 },
-                        { "production-science-pack", 1 },
-                        { "utility-science-pack", 1 },
+                        { "automation-science-pack", 3 },
+                        { "logistic-science-pack", 3 },
+                        { "chemical-science-pack", 3 },
+                        { "production-science-pack", 3 },
+                        { "utility-science-pack", 3 },
+                        { "neural-science-pack", 2 },
```

Comment:
these all should be either all 1 or 4 like the tier of the miners suggest imo

Reply:
No

have u seen
![image](https://user-images.githubusercontent.com/59801736/165779005-c7a04388-b5ce-418f-82ab-52ace4c64a8b.png)

Reply:
Most things I haved looked at were consistenly 1 required science pack with various durations